src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -0,0 +1,381 @@
+| fixel                | Fixel (Fiber bundle element) images. Different voxels can have different numbers of fixels, and therefore a sparse file format is required. This is not specific to Fibre Density derived via CSD. Several diffusion models provide fixel-specific measures (e.g. Ball and stick, CHARMED, CuspMFM, Myelin content). |
+| ODF                  | Orientation distribution function                                                                                                                                                                                                                                                                                    |
+| PDF                  | Diffusion propagator                                                                                                                                                                                                                                                                                                 |
+| FOD                  | Fiber orientation distribution (Sometimes known as fODF)                                                                                                                                                                                                                                                             |
+| EVECS                | Eigenvectors of a model that has eigenvectors (such as DTI)                                                                                                                                                                                                                                                          |
+| PEAKS                | Directions of ODF maxima on the sphere.                                                                                                                                                                                                                                                                              |
+
+## Describing tractography
+
+Tractography normally generates one of two primary file types: tractograms or
+NIfTi files containing maps. Tractograms are files containing a collection of
+streamlines, which are objects describing the path of idealized brain fiber
+fascicles (ensembles of neural fibers that travel together). Tractography maps
+contain visitation counts (i.e. 3D spatial histograms) of a collection of
+streamlines but do not contain the actual streamline objects. The main type of
+map represent the diffusion path probability, or how likely it is that diffusion
+between pairs of brain regions. Two tractogram file formats are supported,
+tractography maps should be saved as NIfTIs.

```

Comment:
Not sure if "tractography maps" makes sense; "tractography-derived maps" would be fine, but many tractography approaches do not either natively or intrinsically produce anything that could be stored in NIfTI.